### PR TITLE
docs: adjusting for cleanup reference docs

### PR DESCRIPTION
**Pull request by jesgreco**
This doc learner_view_of_grades.rst is being cleaned up: https://docs.openedx.org/en/latest/educators/references/grading/learner_view_of_grades.html

### CODE REVIEW

File: source/educators/references/grading/gradebook_assignment_types.rst [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 
-******************************
+##########################
 Gradebook Assignment Types
-******************************
+##########################
 
 .. tags:: educator, reference
 
+.. list-table::
```
Comment (jswope00) [inline source/educators/references/grading/gradebook_assignment_types.rst:9]: What is the purpose of an empty table here?

File: source/educators/references/grading/learner_view_of_grades.rst [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 
-********************************************
+##########################################
 The Learner View of Grades (Progress Page)
-********************************************
+##########################################
 
 .. tags:: educator, reference
 
+.. list-table::
```
Comment (jswope00) [inline source/educators/references/grading/learner_view_of_grades.rst:9]: What is the purpose of an empty table here?

I'm not sure what it is supposed to be rendering, and it is created an error in the build.

File: source/educators/references/grading/gradebook_assignment_types.rst (hunk 1 above)
Comment (jesgreco) [reply source/educators/references/grading/gradebook_assignment_types.rst:9]: This is for adding ToC which is not included in the original doc.

Comment (jesgreco) [reply source/educators/references/grading/gradebook_assignment_types.rst]: I'm fixing those errors by deleting the table.

File: source/educators/references/grading/learner_view_of_grades.rst (hunk 2 above)
Comment (jesgreco) [reply source/educators/references/grading/learner_view_of_grades.rst]: Ok! It was for the ToC which was not included in both docs, but apparently it is not needed, and it's causing the error.

**Comment by jswope00** [comment]
@jesgreco This is happening because when you change a reference to an external link, you've got to update the corresponding links.txt (or links.rst, I forget if it could be both) or else you will get build errors. 

And it isn't as simple as just changing something like partners.edx.org to partners.openedx.org. In almost all cases, Open edX won't have corresponding links for things that exist on edx.org. 

So you've got to make a determination if we can replace the link with something that has similar purpose for Open edX. If not, then you've got to decide how you can remove the reference.

File: source/educators/references/grading/learner_view_of_grades.rst (hunk 2 above)
Comment (jesgreco) [reply source/educators/references/grading/learner_view_of_grades.rst]: @jswope00 tomorrow I'll take a look to the error you're mentioning about reference to an external link. If I am not able to fix it, I'll ask you for help.

**Review by sarina — CHANGES_REQUESTED** [review]
This PR contains changes from #615. Pull requests should only contain one set of changes. I will review this when this doesn't contain the style guide.

**Comment by jesgreco** [comment]
I've just reviewed this PR and it ok to be merged. @sarina let me know if you are ok :)

File: source/educators/references/resources_for_course_teams.rst [hunk 3]
```diff
@@ -1,58 +1,58 @@
...
+###################################
+Resources for Open edX Course Teams
+###################################
 
 .. tags:: educator, reference
 
 Course teams include faculty, instructional designers, course staff, discussion
 moderators, and others who contribute to the creation and delivery of courses
-on edx.org or edX Edge.
+on Open edX platform.
```
Comment (sarina) [inline source/educators/references/resources_for_course_teams.rst:9]:
```suggestion
on an Open edX site.
```

File: source/educators/references/resources_for_course_teams.rst [hunk 4]
```diff
@@ -1,58 +1,58 @@
...
 
-edX101: Overview of Creating a Course
-**************************************
+Open edX101: Overview of Creating a Course
+==========================================
 
-The `edX101`_ course is designed to provide a high-level overview of the course
-creation and delivery process using Studio and the edX LMS. It also highlights
-the extensive capabilities of the edX platform.
+The `Open edX101 <https://sandbox.openedx.org/courses/course-v1:OpenedX+01-2024+2024-1/about>`_ course is designed to provide a high-level overview of the course
```
Comment (sarina) [inline source/educators/references/resources_for_course_teams.rst:26]:
```suggestion
The `Open edX Demo Course <https://sandbox.openedx.org/courses/course-v1:OpenedX+01-2024+2024-1/about>`_ course is designed to provide a high-level overview of the course
```

File: source/educators/references/resources_for_course_teams.rst [hunk 5]
```diff
@@ -1,58 +1,58 @@
...
+about using the Open edX platform to deliver educational experiences. These courses
+are available on Open edX platform.
 
 .. contents::
  :local:
  :depth: 1
 
-edX101: Overview of Creating a Course
-**************************************
+Open edX101: Overview of Creating a Course
```
Comment (sarina) [inline source/educators/references/resources_for_course_teams.rst:23]:
```suggestion
Open edX Demo Course
```

We no longer have the "edX101" course. Please change this section to reference the Demo course and explain it briefly (look at the about page on https://sandbox.openedx.org/courses/course-v1:OpenedX+01-2024+2024-1/about for more detail)

File: source/educators/references/resources_for_course_teams.rst [hunk 6]
```diff
@@ -1,58 +1,58 @@
...
 Course teams include faculty, instructional designers, course staff, discussion
 moderators, and others who contribute to the creation and delivery of courses
-on edx.org or edX Edge.
+on Open edX platform.
 
-======================================
-The edX Course Creator Series
-======================================
+**********************************
+The Open edX Course Creator Series
```
Comment (sarina) [inline source/educators/references/resources_for_course_teams.rst:12]: There is no longer a course creator series, so just highlight the Demo course.

File: source/educators/references/resources_for_course_teams.rst [hunk 7]
```diff
@@ -67,29 +67,25 @@ a PDF version, select **v: latest** at the lower right of the page, then select
...
-====================
+*******************
 Wikis and Web Sites
-====================
+*******************
 
-The edX product team maintains public product roadmaps on :ref:`the Open edX
-Portal<The Open edX Portal>` and :ref:`the edX Partner Portal<The edX Partner
-Portal>`.
+The Open edX product team maintains public product roadmaps on `the Open edX Portal <https://openedx.org/>` and `the Open edX Commnity <https://openedx.org/community/>`_.
```
Comment (sarina) [inline source/educators/references/resources_for_course_teams.rst:90]: This is no longer accurate. Please reference the public roadmap at https://github.com/orgs/openedx/projects/4 and how to get involved with defining the roadmap by joining the Product Working Group https://openedx.atlassian.net/wiki/spaces/OEPM/overview

File: source/educators/references/resources_for_open_edx.rst [hunk 8]
```diff
@@ -1,10 +1,10 @@
...
+######################
 Resources for Open edX
-**************************
+######################
 
 .. tags:: educator, reference
 
 Hosting providers, platform extenders, core contributors, and course staff all
-use Open edX. EdX provides release-specific documentation, as well as the
+use Open edX. Open edX platform provides release-specific documentation, as well as the
```
Comment (sarina) [inline source/educators/references/resources_for_open_edx.rst:8]: Please use "Open edX" as an adjective

File: source/educators/references/student_management/enrollment_requirements.rst [hunk 9]
```diff
@@ -19,21 +19,21 @@ information, see :ref:`Report Learners Not Yet Enrolled`.
...
+***************************
 Registration and Enrollment
-*********************************
+***************************
 
 Before a learner can enroll in a course, he or she must complete these steps.
 
 #. Register a user account, which includes supplying a valid email address, on
-   ``www.edx.org``, ``edge.edx.org``, or your implementation of the edX
+   ``www.openedx.org``, or your implementation of the Open edX
```
Comment (sarina) [inline source/educators/references/student_management/enrollment_requirements.rst:29]:
```suggestion
   your implementation of the Open edX
```

**Review by sarina — CHANGES_REQUESTED** [review]

File: source/educators/references/resources_for_course_teams.rst (hunk 7 above)
Comment (jesgreco) [reply source/educators/references/resources_for_course_teams.rst:88]: done

File: source/educators/references/resources_for_open_edx.rst (hunk 8 above)
Comment (jesgreco) [reply source/educators/references/resources_for_open_edx.rst]: Done!

**Review by jesgreco — COMMENTED** [review]
@sarina I've just finished with all your comments!

File: source/educators/references/resources_for_course_teams.rst [hunk 10]
```diff
@@ -1,58 +1,48 @@
...
 
-edX101: Overview of Creating a Course
-**************************************
+Open edX Demo Course
+====================
 
-The `edX101`_ course is designed to provide a high-level overview of the course
-creation and delivery process using Studio and the edX LMS. It also highlights
-the extensive capabilities of the edX platform.
+The `Open edX Demo Course <https://sandbox.openedx.org/courses/course-v1:OpenedX+01-2024+2024-1/about>`_ course is designed to In discuss the tools and features that can leverage your learning program. After providing details about the platform, the course shows the different types of content that can be created, showcase the variety of assessment tools, and explore the different ways that you can engage your learners with social interactions.
```
Comment (sarina) [inline source/educators/references/resources_for_course_teams.rst:18]:
```suggestion
The `Open edX Demo Course <https://sandbox.openedx.org/courses/course-v1:OpenedX+01-2024+2024-1/about>`_ course is designed to discuss the tools and features that you can leverage in your learning program. After providing details about the platform, the course shows the different types of content that can be created, showcase the variety of assessment tools, and explore the different ways that you can engage your learners with social interactions.
```

File: source/educators/references/resources_for_course_teams.rst [hunk 11]
```diff
@@ -1,58 +1,48 @@
...
 
-StudioX: Creating a Course with edX Studio
-*************************************************
+StudioX: Creating a Course with Open edX Studio
```
Comment (sarina) [inline source/educators/references/resources_for_course_teams.rst:20]: Sorry I wasn't specific enough in my previous comments that only the DemoX course exists. Please remove this, there is no Studio X course, or any of the following courses.

File: source/educators/references/resources_for_course_teams.rst [hunk 12]
```diff
@@ -1,58 +1,48 @@
...
+===============================================
 
-After you complete edX101, "StudioX" provides more detail about using Studio
+After you complete Open edX101, "StudioX" provides more detail about using Studio
 to create a course, add different types of content, and configure your course
 to provide an optimal online learning experience.
 
-BlendedX: Blended Learning with edX
-************************************
+BlendedX: Blended Learning with Open edX
```
Comment (sarina) [inline source/educators/references/resources_for_course_teams.rst:27]: Please remove this, there is no Blended X course

File: source/educators/references/resources_for_course_teams.rst [hunk 13]
```diff
@@ -1,58 +1,48 @@
...
+========================================
 
 In "BlendedX" you explore ways to blend educational technology with
 traditional classroom learning to improve educational outcomes.
 
-VideoX: Creating Video for the edX Platform
-*************************************************
+VideoX: Creating Video for the Open edX Platform
```
Comment (sarina) [inline source/educators/references/resources_for_course_teams.rst:33]: Please remove this, there is no Video X course

File: source/educators/references/resources_for_course_teams.rst [hunk 14]
```diff
@@ -67,29 +57,25 @@ a PDF version, select **v: latest** at the lower right of the page, then select
 **PDF**.
 
 .. note:: If you use the Safari browser, be aware that it does not support the
- search feature for the HTML versions of the edX guides. This is a known
+ search feature for the HTML versions of the Open edX guides. This is a known
  limitation.
 
-======
+*****
 Email
```
Comment (sarina) [inline source/educators/references/resources_for_course_teams.rst:64]: Remove this section, it is entirely edx.org specific

**Review by sarina — CHANGES_REQUESTED** [review]

File: source/educators/references/resources_for_course_teams.rst (hunk 10 above)
Comment (jesgreco) [reply source/educators/references/resources_for_course_teams.rst]: Done! Thanks for those edits!

File: source/educators/references/resources_for_course_teams.rst (hunk 11 above)
Comment (jesgreco) [reply source/educators/references/resources_for_course_teams.rst]: done :)

File: source/educators/references/resources_for_course_teams.rst (hunk 13 above)
Comment (jesgreco) [reply source/educators/references/resources_for_course_teams.rst]: done :)

File: source/educators/references/resources_for_course_teams.rst (hunk 14 above)
Comment (jesgreco) [reply source/educators/references/resources_for_course_teams.rst]: done!

**Comment by jesgreco** [comment]
@sarina all changes are done!

**Review by sarina — APPROVED** [review]

**Pull request merged**